vnext/Microsoft.ReactNative/IReactContext.idl

```diff
@@ -11,70 +11,158 @@ import "IReactPropertyBag.idl";
+    "- Use the @.Properties to share native module's data with other components.\n"
+    "- Use the @.Notifications to exchange events with other components.\n"
+    "- Use @.CallJSFunction to call JavaScript functions, and @.EmitJSEvent to raise JavaScript events.\n"
+    "- Use @.UIDispatcher to post asynchronous work in the UI thread.\n"
+    "- Use @.JSDispatcher to post asynchronous work in the JavaScript engine thread.")
+  interface IReactContext
+  {
 #ifndef CORE_ABI
-    DOC_STRING("Get settings snapshot that were used to start the React instance.")
+    DOC_STRING("Gets the settings snapshot that was used to start the React instance.")
     IReactSettingsSnapshot SettingsSnapshot { get; };
 #endif
 
-    DOC_STRING("Properties shared with the @ReactInstanceSettings.Properties. It can be used to share values and state between components.")
+    DOC_STRING(
+      "Gets @IReactPropertyBag shared with the @ReactInstanceSettings.Properties.\n"
+      "It can be used to share values and state between components and the applications.")
     IReactPropertyBag Properties { get; };
 
     DOC_STRING(
-      "Notifications shared with the @ReactInstanceSettings.Notifications. They can be used to exchange events between components. \n"
-      "All subscriptions added to the `IReactContext.Notifications` are automatically removed after the `IReactContext` is destroyed. \n"
-      "The subscriptions added to the `ReactInstanceSettings.Notifications` are kept as long as `ReactInstanceSettings` is alive.")
+      "Gets @IReactNotificationService shared with the @ReactInstanceSettings.Notifications.\n"
+      "It can be used to send notifications events between components and the application.\n"
+      "All notification subscriptions added to the @IReactContext.Notifications are automatically removed "
+      "after the @IReactContext is destroyed.\n"
+      "The notification subscriptions added to the @ReactInstanceSettings.Notifications are kept "
+      "as long as the @ReactInstanceSettings is alive.")
     IReactNotificationService Notifications { get; };
 
     DOC_STRING(
-      "Get the UI thread dispatcher. \n"
+      "Gets the UI thread dispatcher.\n"
       "It is a shortcut for the @ReactDispatcherHelper.UIDispatcherProperty from the @.Properties property bag.")
     IReactDispatcher UIDispatcher { get; };
 
     DOC_STRING(
-      "Get the JS thread dispatcher. \n"
+      "Gets the JavaScript engine thread dispatcher.\n"
       "It is a shortcut for the @ReactDispatcherHelper.JSDispatcherProperty from the @.Properties property bag.")
     IReactDispatcher JSDispatcher { get; };
 
-    DOC_STRING("Get the JavaScript runtime for the running React instance. It can be null if Web debugging is used.")
+    DOC_STRING(
+      "Gets the JavaScript runtime for the running React instance.\n"
+      "It can be null if Web debugging is used.\n"
+      "**Node: do not use this property directly. "
```

Comment:
could you also include the commit that followed up to fix some more typos? #7481

Reply:
it also fixes some broken links :)

Reply:
Yes, I will do. Thank you!